canonical-web-and-design/ubuntu.com
# Fix the loading of smart-quotes

## Done
Renamed the smartquotes file as it was trying to include itself instead of the module in node_modules.

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- Go to http://0.0.0.0:8001/openstack
- Check there are no longer console errors
- Check the subtitle "Canonical’s Charmed OpenStack" has a nicely curled quote
